hack/jenkins/installers/check_install_golang.sh

```diff
@@ -67,20 +67,28 @@ function check_and_install_golang() {
 
 # install_golang takes two parameters version and path to install.
 function install_golang() {
-  echo "Installing golang version: $1 on $2"
-  pushd /tmp >/dev/null
+  local -r GO_VER="$1"
+  local -r GO_DIR="$2/go"
+  echo "Installing golang version: $GO_VER in $GO_DIR"
 
   INSTALLOS=linux
   if [[ "$OSTYPE" == "darwin"* ]]; then
     INSTALLOS=darwin
   fi
+
+  local -r GO_TGZ="go${GO_VER}.${INSTALLOS}-${ARCH}.tar.gz"
+  pushd /tmp
```

Comment:
Maybe we should preserve the >/dev/null? Since otherwise it will print out the current stack which seems unnecessary.

Reply:
Let's keep it. It's just one line in logs, but it gives an extra bit of information about where we are.